### DISCUSSION TITLE
How to load a custom vscode theme

### DISCUSSION THREAD

**Discussion by rubenfiszel** [TypeFox/monaco-languageclient#567] [Q&A]
Hi, i'd like to load a custom vscode theme that I have the json of: https://raw.githubusercontent.com/nordtheme/visual-studio-code/develop/themes/nord-color-theme.json

I'm not sure exactly how to use the the theme override to load this arbitrary json. I saw the updateUserConfiguration but it seems to be by name instead of accepting an url to load from

Note: I don't intend to loading from the github url but the json in our assets folder :)

**Comment by kaisalmen** [comment]
@rubenfiszel you have to define your own extension. E.g. look at the the [package.json of default vscode themes](https://github.com/microsoft/vscode/blob/main/extensions/theme-defaults/package.json) and then build your extension like we do here:
https://github.com/TypeFox/monaco-languageclient/blob/main/packages/examples/src/langium/statemachineClient.ts#L46-L83
